src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeLowRiskCellConfigurator.swift

```diff
@@ -41,7 +41,7 @@ final class HomeLowRiskCellConfigurator: HomeRiskLevelCellConfigurator {
 		self.detectionInterval = detectionInterval
 		super.init(
 			isLoading: false,
-			isButtonEnabled: detectionMode == .manual && manualExposureDetectionState == .possible,
+			isButtonEnabled: manualExposureDetectionState == .possible,
```

Comment:
For example this button. So it used to be enabled when `detectionMode` was the same as the value of `.manual` (which probably is a constant), so meaning "detection mode is manual" (instead of "automatic"). Where does the requirement come from that now this button should be disabled when "detection mode is manual or automatic"?

Reply:
@levino Thank you very much for this question. I think the change above has no effect on the actual UI. `manualExposureDetectionState` now returns `nil` in case `detectionMode` is `.automatic`. Thus `detectionMode == .manual` can be removed as a condition. This PR also comes with tests.